spec/govuk_error/configuration_spec.rb

```diff
@@ -11,20 +11,20 @@
     end
   end
 
-  describe ".should_capture" do
+  describe ".before_send" do
     let(:configuration) { GovukError::Configuration.new(Raven.configuration) }
 
     it "ignores errors if they happen in an environment we don't care about" do
       ClimateControl.modify SENTRY_CURRENT_ENV: "some-temporary-environment" do
         configuration.active_sentry_environments << "production"
-        expect(configuration.should_capture.call(StandardError.new)).to eq(false)
+        expect(configuration.before_send.call(StandardError.new)).to be_falsy
```

Comment:
Any particular reason why we've dropped `eq(false)` to `be_falsy`? Explicit boolean checks tend to be better, as `be_falsy` will pass even if the returned value is 0, which could be surprising.

Reply:
I made this change because the behaviour of `should_capture` is to return a boolean specifically, but `before_send` ordinarily just returns its input, but returns `nil` if it's to be dropped.

That was just the lazy way of testing it, though — I could also change these to look for `nil` or the input explicitly.